crates/nu-command/src/network/http/options.rs

```diff
@@ -97,18 +97,8 @@ impl Command for SubCommand {
                 result: None,
             },
             Example {
-                description: "Get options from example.com, with username and password",
-                example: "http options -u myuser -p mypass https://www.example.com",
-                result: None,
-            },
```

Comment:
I don't think it does any harm to have other generic examples here. So, I'd leave these.

Reply:
Added them back in.

Reply:
Thanks!